app/lib/navigation/state_file_nj_question_navigation.rb

```diff
@@ -41,14 +41,14 @@ class StateFileNjQuestionNavigation < Navigation::StateFileBaseQuestionNavigatio
                                           Navigation::NavigationStep.new(StateFile::Questions::NjMedicalExpensesController),
                                           Navigation::NavigationStep.new(StateFile::Questions::NjHouseholdRentOwnController),
                                           Navigation::NavigationStep.new(StateFile::Questions::NjHomeownerEligibilityController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjTenantEligibilityController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjIneligiblePropertyTaxController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjHomeownerPropertyTaxWorksheetController),
                                           Navigation::NavigationStep.new(StateFile::Questions::NjHomeownerPropertyTaxController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjTenantPropertyTaxWorksheetController),
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjHomeownerPropertyTaxWorksheetController),
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjTenantEligibilityController),
                                           Navigation::NavigationStep.new(StateFile::Questions::NjTenantRentPaidController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjEitcQualifyingChildController),
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjTenantPropertyTaxWorksheetController),
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjIneligiblePropertyTaxController),
```

Comment:
[sand] - we could test the flow after removing these screens from the navigation since they only get navigated to "manually" using `next_step()` anyway. That may also free us from the need to tie the offramp from the Property Tax flow to the question that happens to come next.

Reply:
captured in https://github.com/newjersey/affordability-pm/issues/227